### PR TITLE
Fixing target file link addition in weekly scan.

### PR DESCRIPTION
**Pull request by mohammedzee1000**
Signed-off-by: Mohammed Zeeshan Ahmed <mohammed.zee1000@gmail.com>

### CODE REVIEW

**Comment by dharmit** [comment]
That's not fixing `target_file_link`, it's *removing* `target_file_link`

**Comment by mohammedzee1000** [comment]
No, it is being added later in the code.
(https://github.com/CentOS/container-pipeline-service/pull/573/files#diff-cf61f8b89212d506aaeaa2fe8a175811L123)

File: jenkinsbuilder/weekly-scan.py [hunk 1]
```diff
@@ -112,13 +112,7 @@
...
         # Initializing Database entries
         project, created = Project.objects.get_or_create(
-            name=project_name,
-            target_file_link=form_targetfile_link(
-                entry["git-url"],
-                entry["git-path"],
-                entry["git-branch"],
-                entry["target-file"]
-            )
+            name=project_name
```
[the remark this answers is not in the document]
Comment (mohammedzee1000) [reply jenkinsbuilder/weekly-scan.py]: Understood

File: jenkinsbuilder/weekly-scan.py [hunk 2]
```diff
@@ -111,22 +110,17 @@
...
         )
-        project.save()
+        if not project:
+            print(
+                str.format(
+                    "Skipping {}, as project object was not found",
+                    project_name
+                )
+            )
```
Comment (navidshaikh) [inline jenkinsbuilder/weekly-scan.py]: I could not understand why you would spread print statement over 6 lines..
```
print(str.format("Skipping {}, as project object was not found.", project_name))

was just fine too..

the print format used in PR is actually less-readable than one-line.
```
@mohammedzee1000

Comment (navidshaikh) [inline jenkinsbuilder/weekly-scan.py]: I could not understand why you would spread print statement over 6 lines..
```
print(str.format("Skipping {}, as project object was not found.", project_name))

was just fine too..

the print format used in PR is actually less-readable than one-line.
```
@mohammedzee1000

**Pull request merged**